pkgs/applications/audio/galaxybudsclient/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, buildDotnetModule
+, fetchFromGitHub
+, autoPatchelfHook
+, fontconfig
+, xorg
+, libglvnd
+}:
+
+buildDotnetModule rec {
+  pname = "GalaxyBudsClient";
+  version = "4.5.2";
+
+  src = fetchFromGitHub {
+    owner = "ThePBone";
+    repo = pname;
+    rev = version;
+    hash = "sha256-bnJ1xvqos+JP0KF8Z7mX8/8IozcaRCgaRL3cSO3V120=";
+  };
+
+  projectFile = [ "GalaxyBudsClient/GalaxyBudsClient.csproj" ];
+  nugetDeps = ./deps.nix;
+  dotnetFlags = [ "-p:Runtimeidentifier=linux-x64" ];
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    fontconfig
+  ];
+
+  runtimeDeps = [
```

Comment:
Ah. I see why my PR didn't work now.
How did you ascertain which packages to add to this?

Reply:
`nix-build` is good at complaining 😅
And with the help of `nix-locate lib/package-name --top-level` I managed to find out what packages contained the lib files the package needed!